windows/RNCNetInfoCPP/RNCNetInfo.h

```diff
@@ -74,7 +80,8 @@ namespace winrt::ReactNativeNetInfo::implementation {
         REACT_EVENT(NetworkStatusChanged, L"netInfo.networkStatusDidChange");
         std::function<void(NetInfoState)> NetworkStatusChanged;
 
-        static std::future<NetInfoState> GetNetworkStatus();
+        static std::future<NetInfoState> GetNetworkStatus(std::string const& requestedInterface = "");
+        static IAsyncAction ChainGetNetworkStatus(IAsyncAction previousRequest, std::future<NetInfoState> currentRequest, std::function<void(NetInfoState)> onComplete);

```

Comment:
This doesn't need to be a public method. It doesn't even need to be a static member of the class. It's fine for it to be a free function that lives solely in the .cpp file.